src/main/scala/io/sdkman/changelogs/JMeterMigration.scala

```diff
@@ -2,21 +2,39 @@ package io.sdkman.changelogs
 
 import com.github.mongobee.changeset.{ChangeLog, ChangeSet}
 import com.mongodb.client.MongoDatabase
+import org.bson.Document
 
 @ChangeLog(order = "051")
 class JMeterMigration {
+
+  @ChangeSet(
+    order = "003",
+    id = "003-add_latest_jmeter_candidate_urls",
+    author = "xshyamx"
+  )
+  def migration003(implicit db: MongoDatabase): List[Version] = {
+    List(
+      "5.4.3",
+      "5.4.2"
+    ).map(
+        version =>
+          Version(
+            candidate = "jmeter",
+            version = version,
+            url =
+              s"https://archive.apache.org/dist/jmeter/binaries/apache-jmeter-$version.zip"
+          )
+      )
+      .validate()
+      .insert()
+  }
   @ChangeSet(
-    order = "001",
```

Comment:
Sorry to be a pain, but let's keep the main candidate migration here for reference like we've done everywhere else. We're only removing version migrations for brevity.

Reply:
No problem. I understand the reasoning behind :). Updated the PR